lib/generators/react_on_rails/react_no_redux_generator.rb

```diff
@@ -11,9 +11,9 @@ class ReactNoReduxGenerator < Rails::Generators::Base
       source_root(File.expand_path("templates", __dir__))
 
       def copy_base_files
-        base_js_path = "base/base"
-        base_files = %w[app/javascript/bundles/HelloWorld/components/HelloWorld.jsx]
-        base_files.each { |file| copy_file("#{base_js_path}/#{file}", file) }
+        base_js_path = "base/base/app/javascript"
```

Comment:
This should change to `base/base/client/app`

create a client directory here: https://github.com/shakacode/react_on_rails/tree/master/lib/generators/react_on_rails/templates/base/base